### DISCUSSION TITLE
Optional ReadMe.txt in GEDZIP specification

### DISCUSSION THREAD

**Discussion by emyoulation** [FamilySearch/GEDCOM#166] [Ideas]
A human readable (with NO parsing or layout requirements but with suggestions) plain text file should specified as an optional (but recommended) top-level file in the GEDZIP format. This is in addition to the gedcom.ged file & each local file: FILE.

The file might include a summary of the provenance of the compressed data. The information is something that might be displayed by tools that read, import, or otherwise process the GEDZIP file. 

The purpose of a ReadMe.txt file is to help a person decide if this content is something in which they wish to delve deeper WITHOUT resorting to actually parsing the GEDCOM structure. This Note could be discarded during parsing of the actual GEDCOM data.

A more questionable addition... a top-level optional text file with processing directives for automated processing. Alternatively, it could specify document workflow for humans.

(Document Routing, Review and Approval:
Apart from just assigning a workflow, an automated Document Control system will automate aspects such as routing, review, and approval to ensure that the document is signed-off by the right set of people; if a key authority is not available, the system will delegate the task to the person designated as the back-up.)

**Comment by dthaler** [comment]
GEDCOM committee comment: It is currently ambiguous in existing GEDZIP spec whether additional files not referenced in the main GEDCOM file are permitted or not.
We might move it into the FAQ once this discussion comes to a conclusion.

**Comment by fisharebest** [comment]
This is similar to #12 - we would need to add ReadMe.txt to the list of "do not use" filenames for `<g7:FILE>`.
